ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -361,7 +361,7 @@ public void onResponse(Call call, Response response) throws IOException {
 
               // Otherwise send the data in one big chunk, in the format that JS requested.
               String responseString = "";
-              if (responseType.equals("text")) {
```

Comment:
There are other cases below where we don't handle null responseBody could you make sure those are covered too?